### ISSUE TITLE
Combobox only announces changes when new value is a different string length

### ISSUE THREAD

**Issue by gdkraus** [FreedomScientific/VFO-standards-support#379]
### Summary
For a role="combobox", when it is collapsed, if you use the arrow keys to move through the items, JAWS only announces changes to the value when the string length changes between values.

### Expected result

JAWS should announce all changes to the value.

### Actual result

JAWS only announces changes when item n, and item n+1 have different string lengths.

### Example

https://codepen.io/gdkraus-sas/pen/zYGbgyB

Note, this is not a fully functional combobox. It only implements the features needed to demonstrate the problem.

There are three examples. The first two comboboxes have lists of all the same string length. The third list contains a collection of string of varying lengths.

#### JAWS version and build number
JAWS 2020.2001 .70 ILM

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 80.0.3987.149

**Comment by JAWS-test** [comment]
This problem also occurs with JAWS 2019.
The problem does not occur in Firefox, but it does also occur in Edge (Chromium).

When I use the up arrow key to navigate through list items, JAWS exits forms mode and the forms field.

**Comment by gdkraus** [comment]
I should also add that for my test I had explicitly turned the VPC off since this particular control is found within a web application where the user is supposed to turn their VPC off

**Comment by JAWS-test** [comment]
Please note that your implementation does not comply with the current specification of combobox: https://www.w3.org/TR/wai-aria-1.2/#combobox (role=combobox must be located directly at the input field)

**Comment by gdkraus** [comment]
I actually had my first prototype coded that way and the behavior was exactly the same. I can put it back up if you want.

**Comment by JAWS-test** [comment]
Okay, I get it. Then I think that the mistake should be corrected by Freedom Scientific...

**Comment by JAWS-test** [comment]
The problem is not limited to role=combobox. It has been occurring for many years (with both old and current versions of JAWS) even with input fields that do not have an additional role. Then it also occurs in Firefox. I have often noticed with input fields with autocomplete functionality, when the focus remains in the input field and the value in the field is changed during arrow key navigation. Then, in both Firefox and Chrome, JAWS exits forms mode when two entries are the same length.

**Comment by smhigley** [comment]
I'm running into this same issue, and found a couple interesting things:

1. It's not that the length of the string matters, what matters is the text insertion point changing locations. If the insertion point changes (e.g. moving from the start of the string to the end), the change will be read. If it doesn't move, JAWS does not read the change. The changing string lengths is one way for the cursor to move positions.
2. As JAWS-test mentioned, the combobox role makes no difference to this use case

I have a simplified example showing the bug here: https://jsfiddle.net/thpxs4my/show

**Comment by ThiloVoigt555** [comment]
I am also getting the same issue.

Noteworthy is, that wai-aria-1.1 recommends opening the attached dropdown when using a combobox with arrow keys:
https://www.w3.org/TR/wai-aria-1.1/#combobox
